static/js/recent_topics.js

```diff
@@ -498,6 +520,42 @@ function topic_sort(a, b) {
     return -1;
 }
 
+function is_topic_visible_to_user(topic_row) {
+    const scroll_container = $("#recent_topics_table .table_fix_head");
+    // We add around 30px so as Thead height where the topic is not visible.
```

Comment:
"... 30px to account for Thead height where the topic is not visible"?

Reply:
Though one better way to do this is `const thead_height = 30` (or some calculation reading the DOM) and then just use that -- that way you can just read the code, rather than needing a comment.